src/types.ts

```diff
@@ -233,6 +233,122 @@ export enum Key {
   SingleQuote = 222,
 }
 
+export enum KeyValue {
+  Backspace = 'Backspace',
+  Tab = 'Tab',
+  Enter = 'Enter',
+  Shift = 'Shift',
+  Control = 'Control',
+  Alt = 'Alt',
+  Pause = 'Pause',
+  CapsLock = 'CapsLock',
+  Escape = 'Escape',
+  Space = ' ',
+  PageUp = 'PageUp',
+  PageDown = 'PageDown',
+  End = 'End',
+  Home = 'Home',
+  ArrowLeft = 'ArrowLeft',
+  ArrowUp = 'ArrowUp',
+  ArrowRight = 'ArrowRight',
+  ArrowDown = 'ArrowDown',
+  Insert = 'Insert',
+  Delete = 'Delete',
+  Key0 = '0',
+  Key1 = '1',
+  Key2 = '2',
+  Key3 = '3',
+  Key4 = '4',
+  Key5 = '5',
+  Key6 = '6',
+  Key7 = '7',
+  Key8 = '8',
+  Key9 = '9',
+  Keya = 'a',
+  Keyb = 'b',
+  Keyc = 'c',
+  Keyd = 'd',
+  Keye = 'e',
+  Keyf = 'f',
+  Keyg = 'g',
+  Keyh = 'h',
+  Keyi = 'i',
+  Keyj = 'j',
+  Keyk = 'k',
+  Keyl = 'l',
+  Keym = 'm',
+  Keyn = 'n',
+  Keyo = 'o',
+  Keyp = 'p',
+  Keyq = 'q',
+  Keyr = 'r',
+  Keys = 's',
+  Keyt = 't',
+  Keyu = 'u',
+  Keyv = 'v',
+  Keyw = 'w',
+  Keyx = 'x',
+  Keyy = 'y',
+  Keyz = 'z',
+  KeyA = 'A',
+  KeyB = 'B',
+  KeyC = 'C',
+  KeyD = 'D',
+  KeyE = 'E',
+  KeyF = 'F',
+  KeyG = 'G',
+  KeyH = 'H',
+  KeyI = 'I',
+  KeyJ = 'J',
+  KeyK = 'K',
+  KeyL = 'L',
+  KeyM = 'M',
+  KeyN = 'N',
+  KeyO = 'O',
+  KeyP = 'P',
+  KeyQ = 'Q',
+  KeyR = 'R',
+  KeyS = 'S',
+  KeyT = 'T',
+  KeyU = 'U',
+  KeyV = 'V',
+  KeyW = 'W',
+  KeyX = 'X',
+  KeyY = 'Y',
+  KeyZ = 'Z',
+  Select = 'Select',
+  Multiply = 'Multiply',
+  Add = 'Add',
+  Subtract = 'Subtract',
+  Decimal = 'Decimal',
+  Divide = 'Divide',
+  F1 = 'F1',
+  F2 = 'F2',
+  F3 = 'F3',
+  F4 = 'F4',
+  F5 = 'F5',
+  F6 = 'F6',
+  F7 = 'F7',
+  F8 = 'F8',
+  F9 = 'F9',
+  F10 = 'F10',
+  F11 = 'F11',
+  F12 = 'F12',
+  NumLock = 'NumLock',
+  ScrollLock = 'ScrollLock',
+  Semicolon = 'Semicolon',
+  Equals = 'Equals',
+  Comma = 'Comma',
+  Dash = 'Dash',
+  Period = 'Period',
+  ForwardSlash = '/',
+  GraveAccent = '`',
+  OpenBracket = '(',
+  BackSlash = '\\',
```

Comment:
Does this need to be escaped? The event will return `\` and this won't match it.